fml/message_loop_impl.cc

```diff
@@ -145,4 +145,8 @@ TaskQueueId MessageLoopImpl::GetTaskQueueId() const {
   return queue_id_;
 }
 
+fml::RefPtr<MessageLoopImpl> MessageLoopImpl::GetLoopImpl(MessageLoop& message_loop) {
```

Comment:
We talked offline. It will be nice to not expose:

1. `MessageLoopImpl::GetLoopImpl`
2. `MessageLoopFuchsia::FuchsiaLoopForMessageLoop`

And get the `async::Loop` from the platform thread using `async::Loop loop(&kAsyncLoopConfigAttachToCurrentThread)`. Once we acquire the `TaskRunners` we can assert that things are happening on the platform thread.